.circleci/config.yml

```diff
@@ -265,11 +263,13 @@ jobs:
       EMAIL: "circleci@localhost"
       GIT_COMMITTER_NAME: "Circle CI"
       GIT_AUTHOR_NAME: "Circle CI"
+      EDGE_BUILD_BRANCH: "D8NID-edge"
```

Comment:
Opted for clarity over security through obscurity here. It's no stretch of the imagination to see which code branch provides the edge build in a public repo, so it's easier setting that variable value here rather than injecting it through Circle CI's env vars options which, annoyingly, you can't see the value of once they're set.